### PR TITLE
AlignDoubleArrowFixer, AlignEqualsFixer - add integration tests

### PR DESCRIPTION
**Pull request by keradus**
Add tests for #1419

### CODE REVIEW

File: Symfony/CS/Tests/AbstractIntegrationTest.php [hunk 1]
```diff
@@ -64,7 +64,7 @@ public static function setUpBeforeClass()
 
     public static function tearDownAfterClass()
     {
-        unlink(static::getTempFile());
+        @unlink(static::getTempFile());
```
Comment (keradus) [inline Symfony/CS/Tests/AbstractIntegrationTest.php]: If no tests were run then unlink will trigger non-existing file error.

cc @SpacePossum

File: Symfony/CS/Tests/AbstractIntegrationTest.php [hunk 2]
```diff
@@ -236,7 +236,7 @@ protected function getFixersFromConfig($fileName, $config)
         }
 
         $fixers = array();
-        for ($i = count(self::$builtInFixers) - 1; $i >= 0; --$i) {
+        for ($i = 0, $limit = count(self::$builtInFixers); $i < $limit; ++$i) {
```
Comment (keradus) [inline Symfony/CS/Tests/AbstractIntegrationTest.php]: We must not change the order of fixers. `Fixer::getFixers()` will sort fixers, so we must use sorted version.

cc @SpacePossum

File: Symfony/CS/Tests/AbstractIntegrationTest.php (hunk 1 above)
Comment (SpacePossum) [inline Symfony/CS/Tests/AbstractIntegrationTest.php]: but how is that possible, if the .. oh I see, :+1: thanks for fixing :)

File: Symfony/CS/Tests/AbstractIntegrationTest.php (hunk 2 above)
Comment (SpacePossum) [inline Symfony/CS/Tests/AbstractIntegrationTest.php]: good catch :+1:

**Comment by SpacePossum** [comment]
:+1:

**Pull request merged**